src/main/java/sigma/TaskList.java

```diff
@@ -0,0 +1,88 @@
+package sigma;
+import sigma.task.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Class that wraps around the list of Tasks with added operations
+ *
+ * @author Qiao Yi
+ */
+public class TaskList {
+    private static List<Task> items;
+
+    /**
+     * Constructor for a TaskList
+     */
+    public TaskList() {
+        items = new ArrayList<>();
+    }
```

Comment:
Some personal suggestions, not relevant to code styles stuff. I think you can make the constructor become private, so that others cannot use the default constructor, then you can just initialize the items when declare. I think your current implementation will get problem if I call other functions before construct a TaskList, since other functions are static, so this condition may happen.

After changing like this,
```
private static List<Task> items = new ArrayList<>();

private TaskList() {
    // do nothing
    // just prevent other call this constructor 
}
```